includes/razorpay-webhook.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
+
+require_once __DIR__.'/../razorpay-sdk/Razorpay.php';
+use Razorpay\Api\Api;
+
+class RZP_Webhook 
+{
+    function __construct()
+    {
+        $this->auto_capture_webhook();
+    }
+
+    function auto_capture_webhook()
+    {
+        $razorpay = new WC_Razorpay();
+
+        $post = file_get_contents('php://input');
+
+        $data = json_decode($post, true);
+
+        if ($razorpay->enable_webhook == 'yes' && !empty($data['event']))
+        {   
+            // if payment.authorized webhook is enabled, we will update woocommerce about captured payments
+            if ($data['event'] == "payment.authorized")
```

Comment:
Check if order is paid at WooCommerce end before running this code. For most payments, that will be the case.